api/tests/tooling/domain-builder/factory/build-assessment-result.js

```diff
@@ -29,3 +30,113 @@ module.exports = function buildAssessmentResult({
     competenceMarks,
   });
 };
+
+buildAssessmentResult.validated = function({
+  id = 123,
+  pixScore = 31,
+  emitter = 'PIX-ALGO',
+  commentForJury = 'Comment for Jury',
+  commentForCandidate = 'Comment for Candidate',
+  commentForOrganization = 'Comment for Organization',
+  createdAt = new Date('2018-01-12T01:02:03Z'),
+  juryId = 1,
+  assessmentId = 456,
+  competenceMarks = [],
+} = {}) {
+  return buildAssessmentResult({
+    id,
+    pixScore,
+    status: assessmentResultStatuses.VALIDATED,
+    emitter,
+    commentForJury,
+    commentForCandidate,
+    commentForOrganization,
+    createdAt,
+    juryId,
+    assessmentId,
+    competenceMarks,
+  });
+};
```

Comment:
Est-ce qu'on a besoin de repréciser tous les arguments par défaut si c'est les mêmes à chaque fois ?

```suggestion
buildAssessmentResult.validated = function(props) {
  return buildAssessmentResult({
    ...props,
    status: assessmentResultStatuses.VALIDATED,
  })
}
```

Reply:
Ah non t'as raison ! j'ai glissé, merci je corrige